### PR TITLE
Wj curate talks page

### PR DESCRIPTION
**Pull request by wjohnson85**
Add a layout and some animations to the talks page

Link to gif of animation: https://share.getcloudapp.com/QwuAyRYv

summer

### CODE REVIEW

**Comment by MaggieAppleton** [comment]
The hover animation is a nice touch, but I think it's best if we keep hover interactions consistent across the site. Anytime someone hovers over a resource (of any kind: course, podcast, talk, article) it should behave the same way.

At the moment we either have no visual indication the element is being hovered over (not great), or the resource title changes from black to blue:
Screen Recording 2021-07-12 at 07 53 37 pm 2021-07-12 19_59_00

I'd vote we remove the hover animation here for the moment, but open up a separate exploration of adding consistent hover animations to resources across the site. We can still use framer motion if that's the right tech choice to get the job done.

**Comment by wjohnson85** [comment]
I [removed framer motion](https://github.com/eggheadio/egghead-next/pull/761/commits/7329d88fb83567af80c97a35ad0f1cf049187eca)

File: src/pages/talks/index.tsx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                      </div>
+                    </a>
+                  </Link>
+                </div>
+              )}
+              <Link href={talk.path}>
+                <a>
+                  <h2 className="md:text-2xl text-xl font-bold leading-tighter">
+                    {talk.title}
+                  </h2>
```
Comment (zacjones93) [inline src/pages/talks/index.tsx:49]:
```suggestion
                  <h2 className="md:text-2xl text-xl font-bold leading-tighter hover:text-blue-600 dark:hover:text-blue-300">
                    {talk.title}
                  </h2>
```

I believe we want to keep the hover effect of the title changing to blue

**Review by dealingwith — APPROVED** [review]
@MaggieAppleton We're keeping the title text hover style here, but we still need to make hover styles consistent across the entire site. Leaving that ball in your court :)

**Pull request merged**